src/types/libdef/npm/jest_v25.x.x.js

```diff
@@ -901,7 +901,7 @@ type JestObjectType = {
 type JestSpyType = { calls: JestCallsType, ... };
 
 type JestDoneFn = {|
- (): void,
+ (error?: Error): void,
```

Comment:
The change has been contributed to flow-typed: https://github.com/flow-typed/flow-typed/pull/3844 so there will be no problem updating the libdef later.